app/src/main/res/value-it/strings.xml

```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Could you please also translate the files `app/src/floss/res/values/strings.xml` and `app/src/google/res/values/strings.xml`?
Create a copy into the `app/src/{floss,google}/res/values-it/` directory as you did with this main file.

Reply:
as soon as I find the time I provide

Reply:
Sure. Take your time. Those are only few lines, though.